Packs/ServiceNow/Integrations/ServiceNowEventCollector/README.md

```diff
@@ -0,0 +1,61 @@
+Use this integration to fetch audit logs from Service Now as Cortex XSIAM events.
+This integration was integrated and tested with Vancouver version of ServiceNow API.
+
+## Configure ServiceNow Event Collector on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for ServiceNow Event Collector.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description**                                                                          | **Required** |
+    | --- |------------------------------------------------------------------------------------------| --- |
+    | ServiceNow URL | ServiceNow URL in the format https://company.service-now.com/                            | True |
+    | Username |                                                                                          | True |
+    | Password |                                                                                          | True |
+    | Client ID |                                                                                          | False |
+    | Client Secret |                                                                                          | False |
+    | ServiceNow API Version (e.g. 'v1') |                                                                                          | False |
+    | Use OAuth Login | Select this checkbox if to use OAuth 2.0 authentication. See \(?\) for more information. | False |
+    | Maximum number of events per fetch | Default value is 1000                                                                    | False |
+    | Events Fetch Interval |                                                                                          | False |
+    | Trust any certificate (not secure) |                                                                                          | False |
+    | Use system proxy settings |                                                                                          | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
+
+## Commands
+
+You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+
+### service-now-get-audit-logs
+
+***
+Returns audit logs events extracted from ServiceNow. This command is used for developing/debugging and is to be used with caution, as it can create events, leading to events duplication and exceeding the API request limitation.
```

Comment:
```suggestion
Returns audit logs events extracted from ServiceNow. This command is used for developing/debugging and is to be used with caution, as it can create events, leading to event duplication and exceeding the API request limitation.
```